### DISCUSSION TITLE
wrong lan IP addr in openwrt script

### DISCUSSION THREAD

**Discussion by raenye** [tteck/Proxmox#1459] [Ideas]
https://github.com/tteck/Proxmox/blob/2a80ed71842ddadbbdc98530c069a58704ad986b/vm/openwrt.sh#L487

Should be `192.168.2.1` IMHO.

**Comment by tteck** [comment]
I agree... changed

**Reply by raenye** [reply]
Thanks.
May I also suggest the following changes:
1. Switch the order of wan and lan bridges. Typical OpenWrt devices have `eth0` for lan and `eth1` for wan;
2. Enable qemu guest agent in proxmox and install the `qemu-ga` OpenWrt package.

**Reply by tteck** [reply]
If you find yourself with some spare time, please consider submitting a pull request (PR) incorporating the suggested changes.

**Reply by raenye** [reply]
Change 1 is #1460.
I don't know enough about ProxMox and/or your scripts to do Change 2.
